fcp/src/containment/fsio/tools.py

```
@@ -1,15 +1,21 @@
+from pantograph import Server
 from containment.structures import LakeResponse
 
 CMD = ["lake", "exe", "check"]
 UP = ".."
 LAKE_DIR = Path.cwd() / UP / "imp"
+LEAN_PATH = subprocess.run(
+    ["lake", "env", "printenv", "LEAN_PATH"],
+    text=True,
+    capture_output=True,
+    cwd=LAKE_DIR,
+).stdout.strip()
```

Pylint (3.3.7)

[warning] 'subprocess.run' used without explicitly defining the value for 'check'.

(W1510)